src/libraries/System.Formats.Tar/src/System.Formats.Tar.csproj

```diff
@@ -0,0 +1,78 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

Comment:
Is it expected that Browser will not be able to use this library? cc @ericstj

Also this library is missing a default tfm, so you may want to change the `$(NetCoreAppCurrent)-Unix` tfm to be the default ridless tfm.

Reply:
This will need to have a platform agnostic build.  I suspect we'd face an error validating the shared framework if we don't do that (I hope!) ref without source.

Reply:
Do I need to push another change for this backport, or can we leave this for a future PR in main?